docs/content/ingestion/batch-delta-ingestion.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: doc_page
+---
+
+This document assumes the reader has understanding of [hadoop batch ingestion](batch-ingestion.html). You can use hadoop batch-ingestion to do re-indexing and delta-ingestion as well.
+
```

Comment:
I think it is useful to say that updates are best done through batch ingestion, and this section will focus on Hadoop based batch ingestion.